third_party/islet/include/islet.h

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2023 Samsung Electronics Co., Ltd. All Rights Reserved.
+
+#pragma once
+
+#include <cstdarg>
+#include <cstdint>
+#include <cstdlib>
+#include <ostream>
+#include <new>
+
+constexpr static const int ISLET_E_SUCCESS = 0;
+
+constexpr static const int ISLET_E_FAILURE = -1;
+
+constexpr static const int ISLET_E_INPUT = -2;
+
+constexpr static const int ISLET_E_WRONG_REPORT = -3;
+
+constexpr static const int ISLET_E_WRONG_CLAIMS = -4;
+
+constexpr static const int ISLET_E_NOT_SUPPORTED = -5;
+
+extern "C" {
+
+/// Get an attestation report(token).
+///
+/// # Note
+/// This API returns hard coded report to simuate temporarily.
+/// `User data` could be used as nonce to prevent reply attack.
+int islet_attest(const unsigned char *user_data,
+                 int user_data_len,
+                 unsigned char *report_out,
+                 int *report_out_len);
+
+/// Verify the attestation report and returns attestation claims if succeeded.
+int islet_verify(const unsigned char *report,
+                 int report_len,
+                 unsigned char *claims_out,
+                 int *claims_out_len);
+
+/// Parse the claims with the given title and returns the claim if succeeded.
+int islet_parse(const char *title,
+                const unsigned char *claims,
+                int claims_len,
+                unsigned char *value_out,
+                int *value_out_len);
+
+/// Print all claims including Realm Token and Platform Token.
+void islet_print_claims(const unsigned char *claims, int claims_len);
+
+/// Seals the plaintext given into the binary slice
+///
+/// # Note
+/// This API seals with hard coded key to simuate temporarily.
```

Comment:
Typo: 'simulate'.

I think a rephrased message would be better:

```
This API currently seals with a hard-coded key, to simulate seal operation. 
In future, this will be finalized to support _<fill-in-the-blank>_ keys.
```

Replace the _<fill-in-the-blank>_  with some term better describing what you intend to do in the future.

Hope this rewrite makes sense to you ...

Reply:
Thanks for suggestion! I reflected it.